app/services/qualified_teachers_api_sender.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+class QualifiedTeachersApiSender
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  SUCCESS_CODES = %w[204].freeze
+
+  attribute :participant_outcome_id
+
+  validates :participant_outcome_id, presence: { message: I18n.t("errors.participant_outcomes.missing_participant_outcome_id") }
+  validates :participant_outcome, presence: { message: I18n.t("errors.participant_outcomes.missing_participant_outcome") }
+  validate :not_already_sent
+
+  def call
+    return if invalid?
```

Comment:
chatted in person, we still need to avoid this as it might fail silently and re-queue, so we might need to move up: `set_sent_to_qualified_teachers_api_at`